spec/app_board_view_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+
+RSpec.describe "Board view", type: :feature do
+  before do
+    Capybara.app = Sinatra::Application
+    stub_const("CACHE_EXPIRY_TIMEOUT", 0)
+  end
+
+  context "get '/board/dxw'" do
+    it "shows today's date" do
+      stub_gcal_request(gcal_fake_response([]))
+      visit "/board/dxw"
+
+      expect(page).to have_selector("#datetime_date", text: Date.today.strftime("%A, %B %e"))
+    end
+    it "shows the Zoom call rooms A, B, and C" do
+      stub_gcal_request(gcal_fake_response([]))
+      visit "/board/dxw"
+
+      expect(page).to have_content("Zoom Call A")
+      expect(page).to have_content("Zoom Call B")
+      expect(page).to have_content("Zoom Call C")
+    end
+  end
+
+  context "get '/board/hoxton" do
```

Comment:
These tests feel strange to me, because they're not really testing that the _application_ is working as intended, but instead they're testing that the boards are configured as expected. If we renamed the Zoom rooms to Alpha, Beta and Gamma then although the application would still be correct the tests would fail.

Again, the fix would be to have a `test-boards.json` which contains exactly what we would expect to see in testing, and then basing tests around that.

Reply:
I agree those tests are a bit brittle. I think having that `json` file you mentioned is a much better idea. I'll see what I can do. Thanks for the suggestion!

Reply:
I've done these changes. I created a `test_boards.json` inside my `spec` folder. I had to create a `test_rooms.json` as well, as they are dependant on each other. Now, it wouldn't matter if the meeting rooms get renamed in the future (or even if Hoxton office does not exist anymore!). Let me know if this is close to what you expected. 
I also added assertions that meeting rooms from other boards will not be displayed.

Reply:
This is a much stronger test, even though it doesn't actually look much different!